app/enterprise/2.1.x/brain-immunity/overview.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Kong Brain & Kong Immunity Overview
+---
+
+**Kong Brain** and **Kong Immunity** help automate the entire API and service development life cycle. By automating processes for configuration, traffic analysis and the creation of documentation, Kong Brain and Kong Immunity help organizations improve efficiency, governance, reliability and security. Kong Brain automates API and service documentation and Kong Immunity uses advanced machine learning to analyze traffic patterns to diagnose and improve security.
+
+![Kong Brain and Kong Immunity](/assets/images/docs/ee/brain_immunity_overview.png)
+
+### About Kong Brain
+Kong Brain automates tasks, increases visibility, and ensures consistency across complex architectures. Other features include:
+
+#### Visually Map Services
+Get a high-level view of your architecture with a real-time visual service map. Analyze inter-service dependencies across teams and platforms to improve governance and minimize risk. See [Using the Service Map](/enterprise/latest/brain-immunity/service-map/).
+
+#### Auto-generate Documentation
+Standardize docs with hands-free creation of OpenAPI spec files for all your services. Transition seamlessly to spec-first development to increase collaboration and developer efficiency. See [Auto-Generated Specs](/enterprise/latest/brain-immunity/auto-generated-specs/).
+
+#### Streamline Issue Resolution
+Contextualize issues and take immediate action by surfacing Kong Immunity anomaly alerts in the service map. Rapidly identify issues impacting services to reduce risk and minimize downtime.
+
+
+### About Kong Immunity
+Kong Immunity autonomously identifies service issues with machine learning-powered anomaly detection. Other features include:
+
+#### Create a Baseline for Healthy Traffic
+Traffic patterns provide a window into the behavior and performance of services under different conditions. To understand your existing traffic patterns, Kong Immunity ingests data flowing through the Kong data plane to create a baseline for healthy traffic. As anomalies are detected and addressed through changes to the Kong Enterprise configuration, Kong Immunity continuously adapts this baseline. Kong Immunity uses the Median Average Deviation (MAD) of traffic to detect anomalies. The dataset used to calculate the expected MAD continuously updates based on the last week of data available.
```

Comment:
We only use MAD for traffic count anomalies.  If you need to specify some more how the other alerts are generated, most of it are statistical methods (including MAD).  But I would also talk to marketing/sales first about that because they have been strongly insinuating that Machine Learning is behind Immunity, and if you say "statistical methods" are what power Immunity, then the strong insinuation of Machine Learning is debunked immediately.